### PR TITLE
Feature input system

### PR DESCRIPTION
**Pull request by STACHY225**
Still gotta check some funcs, but they shall work

### CODE REVIEW

File: Engine-Editor/GameJam/Assets/Scripts/GameJam/Source/Players/CirclePlayer.cs [hunk 1]
```diff
@@ -48,6 +48,13 @@ public void OnCollisionExit(Collision2D collision)
             collision.otherEntity.GetComponent<SpriteRendererComponent>().color = Color.brown;
         }
 
+        public void OnKeyPress(KeyCode keyCode)
+        {
+            DebugConsole.Log("Key pressed "+keyCode, DebugConsole.LogType.Info);
+            // Input.SetClipboardContent("Nacisnales " + keyCode.ToString());
```
Comment (miciex) [inline Engine-Editor/GameJam/Assets/Scripts/GameJam/Source/Players/CirclePlayer.cs]: Delete comments if the purpose of their existance was testing.

**Review by miciex — COMMENTED** [review]
In the next Input System update add possibility in scripts to disable certain or all inputs for chosen or all scripts

File: Engine-Editor/GameJam/Assets/Scripts/GameJam/Source/Players/CirclePlayer.cs (hunk 1 above)
Comment (STACHY225) [reply Engine-Editor/GameJam/Assets/Scripts/GameJam/Source/Players/CirclePlayer.cs]: Done #26

**Review by miciex — APPROVED** [review]
Required changes were made

**Pull request merged**